### PR TITLE
adds content :tm: 

### PR DESCRIPTION
**Pull request by octoshrimpy**
Adds a handful of apps I use on near-daily basis, as well as a couple places to find more TUI and otherwise lightweight app alternatives.

### CODE REVIEW

**Comment by octoshrimpy** [comment]
fixed outustanding issues with one app update and one request

File: README.md [hunk 1]
```diff
@@ -44,55 +44,63 @@ List of projects that provide terminal user interfaces
...
 - [Mutt](https://gitlab.com/muttmua/mutt) Email client
 - [sclack](https://github.com/haskellcamargo/sclack) Slack terminal client
+- [siggo](https://github.com/derricw/siggo) A terminal ui for signal-cli, written in Go
 - [Slack-term](https://github.com/erroneousboat/slack-term) Slack client for your terminal
 - [TelegramTUI](https://github.com/bad-day/TelegramTUI) Telegram client
 - [Weechat](https://weechat.org/) Extensible chat client
 
 ## <a name="miscellaneous"></a>Miscellaneous
 
+- [diary](https://github.com/actuday6418/Diary) A diary app written in Rust that ecrypts both text and file data, and can decrypt and build a rich HTML representation of your diary when required.
```
Comment (lodenrogue) [inline README.md:96]: ecrypts should be encrypts

**Review by lodenrogue — CHANGES_REQUESTED** [review]
Everything else looks good

File: README.md (hunk 1 above)
Comment (octoshrimpy) [reply README.md]: thanks!  taken care of.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ List of projects that provide terminal user interfaces
 - [Miscellaneous](#miscellaneous)
 - [Multimedia](#multimedia)
 - [Productivity](#productivity)
-- [Web](#web)
+- [Web](#web)e
```
Comment (rothgar) [inline README.md:15]: extra `e`

**Comment by rothgar** [comment]
Thanks for all the links and formatting. Just one comment to update.

File: README.md (hunk 2 above)
Comment (octoshrimpy) [reply README.md]: don't know how I missed that haha! resolved!

**Pull request merged**